### DISCUSSION TITLE
Error during deployment

### DISCUSSION THREAD

**Discussion by legyul** [alshedivat/al-folio#2759] [General]
Firstly, thank you very much for being able to use this template.

During the deployment manually, I got the following errors.

Imagemagick: Command returned pid 14725 exit 127 with error sh: convert: command not found 
[NbConvertApp] Converting notebook /var/folders/wx/xs4bgqb95lb8vc6wdx8pzw2m0000gn/T/jekyll-jupyter-notebook20241009-14714-99nm50.ipynb to html
[NbConvertApp] Converting notebook /var/folders/wx/xs4bgqb95lb8vc6wdx8pzw2m0000gn/T/jekyll-jupyter-notebook20241009-14714-r0ybut.ipynb to html
[NbConvertApp] WARNING | Alternative text is missing on 17 image(s).

How can I fix this?

**Comment by george-gca** [comment]
Can you post the full stack trace or link to your repo? This part of the log is not useful at all.

**Reply by legyul** [reply]
https://github.com/namdarine/namdarine.github.io
this is my repo.

**Reply by george-gca** [reply]
Sorry for the delay. Just checked your repo and I think everything is fine already?
